package.json

```
@@ -123,8 +121,7 @@
     "remark-extract-anchors": "1.1.1",
     "remark-frontmatter": "^3.0.0",
     "remark-gfm": "^1.0.0",
-    "remark-html": "^13.0.1",
-    "remark-loader": "^4.0.0",
+    "remark-html": "^13.0.0",
     "remark-refractor": "git://github.com/montogeek/remark-refractor.git",
```

`git` protocol is incorrect for other contributors.

```suggestion
    "remark-refractor": "montogeek/remark-refractor",
```

---

Thanks, I didn't know it works in this way.